build.sbt

```diff
@@ -227,27 +226,6 @@ lazy val attachProtobufDescriptorSets = Seq(
   Compile / unmanagedResourceDirectories ++= (Compile / PB.protoSources).value
 )
 
-lazy val `java-valueentity-shopping-cart` = project
-  .in(file("samples/java-valueentity-shopping-cart"))
-  .dependsOn(sdk, testkit % IntegrationTest)
-  .enablePlugins(AkkaGrpcPlugin, IntegrationTests, LocalDockerImage)
-  .settings(
-    name := "java-valueentity-shopping-cart",
-    libraryDependencies ++= Seq(
-        "ch.qos.logback" % "logback-classic" % LogbackVersion,
-        "com.novocode" % "junit-interface" % JUnitInterfaceVersion % IntegrationTest
-      ),
-    Compile / mainClass := Some("valueentity.shoppingcart.Main"),
-    // Akka gRPC only for IntegrationTest
-    Compile / akkaGrpcGeneratedSources := Seq.empty,
-    Compile / javacOptions ++= Seq("-encoding", "UTF-8", "-source", "11", "-target", "11"),
-    testOptions += Tests.Argument(TestFrameworks.JUnit, "-q", "-v", "-a"),
-    IntegrationTest / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java),
-    IntegrationTest / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Client),
-    IntegrationTest / PB.protoSources ++= (Compile / PB.protoSources).value
-  )
-  .settings(attachProtobufDescriptorSets)
-
```

Comment:
Maybe we should keep a minimal sbt build just to run the check headers and javafmt.

Reply:
That won't work without some extra tweaks. 

sbt can't compile projects that moved to codegen, so we will need to exclude the projects from compile/test and only have then running the checkers. 

I will defer this since not extremely important.